src/main/java/smokefree/aws/rds/secretmanager/SmokefreeConstants.java

```diff
@@ -32,4 +32,9 @@
 	public static final Long MAXIMUM_PLAYGROUNDS_DISTANCE = 100L;
 	public static final Long MAXIMUM_PLAYGROUNDS_ALLOWED = 1000L;
```

Comment:
Just a question: Why do you use a Long and not an Integer?

Reply:
I did not think specifically between Long and Integer at this context.
Yeah, It would be good If Integer is used considering memory usage.

Can I fall back to Integer?

Reply:
whatever. Memory is not issue, neither is the max value of these constants. It's more that I am used to defaulting to int / Integer unless I need to really big number.
Your choice, but be consistent over all of the constants

Reply:
Okay. I will keep Integer and maintain consistency.